### ISSUE TITLE
The range or unit of the output result

### ISSUE THREAD

**Issue by LiuYi-Up** [bardiadoosti/HOPE#32]
Hi Bardia Doosti! Thank you for your excellent work and willing to share it. Recently I am trying to predic the 21 key points of hand, but now I have some questions:
1. Is the unit based on pixels when your network outputs 2D? Or another unit of length?
2. I tried to train on the HO3D dataset because I wanted to predict from the third-person perspective. Are your training steps the same as FHAD for the HO3D dataset?
Looking forward to your reply, thank you.

**Comment by LiuYi-Up** [comment]
Hi Bardia Doosti!I trained it for about 2000 epoch on HO3D, but the 2D results did not seem to be good. They were generally shrunk and were close to the center of the picture.And these are still the performance on the training set where EPE is less than 0.1. And the performance on the validation set or my own data set is worse, with EPE more than 4500. Is it the data set or something else?

**Comment by Liqi255** [comment]
Hi, Liu!  Sorry,  I can't share your questions meaningfully at present, but I don't know if I can ask you some questions, because I'm a green, so there are many basic problems that need to be solved urgently. For example, what is the configuration environment required by this code? Can you tell me your setup scheme? Thank you very much! Looking forward to your reply.

**Comment by LiuYi-Up** [comment]
> Hi, Liu! Sorry, I can't share your questions meaningfully at present, but I don't know if I can ask you some questions, because I'm a green, so there are many basic problems that need to be solved urgently. For example, what is the configuration environment required by this code? Can you tell me your setup scheme? Thank you very much! Looking forward to your reply.

**Comment by LiuYi-Up** [comment]
> Hi, Liu! Sorry, I can't share your questions meaningfully at present, but I don't know if I can ask you some questions, because I'm a green, so there are many basic problems that need to be solved urgently. For example, what is the configuration environment required by this code? Can you tell me your setup scheme? Thank you very much! Looking forward to your reply.

**Comment by LiuYi-Up** [comment]
My configuration environment:
python 3.7.10
torch 1.2.0
cuda V10.0.130

**Comment by Liqi255** [comment]
> My configuration environment:
> python 3.7.10
> torch 1.2.0
> cuda V10.0.130

Your reply is very important to me, thanks and best wishes to you!

**Comment by wangyao0512** [comment]
@LiuYi-Up 您好，请问您的问题解决了吗？我也遇到同样的问题，使用公开数据集训练集上效果很差，预测点大多都集中于图像中间。这是数据输入的问题吗？还是网络结构的问题呢？

**Comment by LiuYi-Up** [comment]
> @LiuYi-Up 您好，请问您的问题解决了吗？我也遇到同样的问题，使用公开数据集训练集上效果很差，预测点大多都集中于图像中间。这是数据输入的问题吗？还是网络结构的问题呢？
您好，我并没有解决这个问题，我的猜想是网络前需要手的检测器crop后输入HOPE-Net，但由于各种原因，我并没有继续实验。

**Comment by LiuYi-Up** [comment]
> @LiuYi-Up 您好，请问您的问题解决了吗？我也遇到同样的问题，使用公开数据集训练集上效果很差，预测点大多都集中于图像中间。这是数据输入的问题吗？还是网络结构的问题呢？

您好，我并没有解决这个问题，我的猜想是网络前需要手的检测器crop后再输入HOPE-Net，但由于各种原因，我并没有继续进行实验。
